caravel/templates/manage_listings.html

```diff
@@ -0,0 +1,45 @@
+{% extends "layout.html" %}
+{% block title %} Manage Listings {% endblock %}
+{% block head %}
+  {{ super() }}
+{% endblock %}
+{% block content %}
```

Comment:
Er, I mean, why have a manage listings page at all? If we have search by seller, can't we just include a link to a ?q=seller:me or something? On the old site, at least, that's how it works — any time it shows a listing that's yours, you get links to the edit pages inline.